cmd/antrea-agent/agent.go

```diff
@@ -294,6 +294,16 @@ func run(o *Options) error {
 		NodePortAddressesIPv6: nodePortAddressesIPv6,
 	}
 
+	if features.DefaultFeatureGate.Enabled(features.SecondaryNetwork) {
```

Comment:
According to the original design for agent, we should firstly initialize the agent, then each feature can start including its own initialize functions.